docs/conf/portland/2018/sponsors/prospectus.rst

```diff
@@ -118,15 +122,18 @@ The **Publisher** package is great for sending a team and getting to know the co
 - Name included in welcome announcement in email newsletters and social media
 - Display 2 promotional (“Swag”) item on the conference swag table (provided by sponsor)
 - Two featured job postings in our Newsletter (2,500 subscribers)
+- A booth at the job fair
```

Comment:
Have we checked that if we get say 8 Publisher sponsors we can fit all the tables in? Main hall, so I guess we can